README.md

```diff
@@ -1,14 +1,52 @@
-# Knative Sample Controller
+# knative/networking
 
-[![GoDoc](https://godoc.org/knative.dev/sample-controller?status.svg)](https://godoc.org/knative.dev/sample-controller)
-[![Go Report Card](https://goreportcard.com/badge/knative/sample-controller)](https://goreportcard.com/report/knative/sample-controller)
+This repository contains the Knative Ingress and Certificate CRDs, as well as
+their conformance tests. These are our extension points to plugin different
+Ingress plugins (Ambassador, Contour, Gloo, Istio, Kong and Kourier), as well as
+different AutoTLS plugins (CertManager and Knative's own HTTP01 challenge
+solver).
 
-Knative `sample-controller` defines a few simple resources that are validated by
-webhook and managed by a controller to demonstrate the canonical style in which
-Knative writes controllers.
 
-To learn more about Knative, please visit our
-[Knative docs](https://github.com/knative/docs) repository.
+# Knative Ingress aka KIngress
 
-If you are interested in contributing, see [CONTRIBUTING.md](./CONTRIBUTING.md)
-and [DEVELOPMENT.md](./DEVELOPMENT.md).
+The Knative Ingress CRD is based on the Kubernetes Ingress CRD, with the
+following additions:
+1. Traffic splitting
+2. Header modification applied to requests based on the traffic split they are
+   assigned to.
+3. Host header rewrite
+4. Traffic redirection predicates based on a regexp based condition on headers.
+
+In addition to these, we previously added Timeout and Retry settings but no
+longer used them and may deprecate these parts in future versions.
+
+See
+https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-serving-component
+for a list of the current supported KIngress implementations.
+
+Check out:
+- [/pkg/apis/networking/v1alpha1/ingress_types.go](/pkg/apis/networking/v1alpha1/ingress_types.go)
+  for more information about the KIngress API spec.
+- [/pkg/apis/networking/v1alpha1/ingress_validation.go](/pkg/apis/networking/v1alpha1/ingress_validations.go)
+  for more information about the validation logic for KIngress API spec.
+- [/test/conformance/ingress/README.md](/test/conformance/ingress/README.md) for
```

Comment:
maybe drop the leading `/` or maybe even the path (outside of the link) to clean this up?  Same below